code/create/dta_dicom2spec_rules.py

```diff
@@ -1,158 +1,115 @@
                 return m
             if "mprage" in prot_parts:
                 return "t1w"
+    return None
 
-        # BEGIN: Additional rule for forrest-structural
-        # TODO: Probably to be moved to some rule enhancement
-        if "st1w" in prot_parts:
-            return "t1w"
-        if "st2w" in prot_parts:
-            return "t2w"
-        if "tof" in prot_parts:
-            return "angio"
-        # END
 
-        if "func" in prot_parts:
-            return "bold"
+def _guess_task(dcm_dict, record):
 
-    # found nothing, but modality isn't necessarily required
-    return None
+    protocol = record.get("ProtocolName", None).lower()
 
-def _guess_task(dcm_dict, record):
-    protocol = record.get("ProtocolName", None)
+    # TODO: do I really need/use this values?
     count_if_same_protocol_but_scanned_earlier = 1
-    #
     exp_flag = False
 
+    # check if protocol is unequal to None
     if protocol:
-        if protocol.lower() in "dta_epi_pmu_rest":
+        if protocol == "dta_epi_pmu_rest":
                return "rest"
-        if protocol.lower() in "DTA_ep2d_bold_Task".lower():
+        if protocol == "DTA_ep2d_bold_Task":
+
             for elem in dcm_dict._dicom_series:
-               if "DTA_ep2d_bold_Task".lower() in elem.get("ProtocolName").lower():
+               if "DTA_ep2d_bold_Task".lower() == protocol:
                   return ( "exp" + get_column( record.get("PatientID") ) )
 
-        if "60dir".lower() in protocol.lower():
+        if "60dir" in protocol:
             return "60dir"
-        if "120dir".lower() in protocol.lower():
+        if "120dir" in protocol:
             return "120dir"
 
-
         import re
-        prot_parts = re.split('_|-|\s', protocol.lower())
+        prot_parts = re.split('_|-|\s', protocol)
+
+	# TODO: is this try/except-block helpful for DTA or just for forrest?
         try:
             idx = prot_parts.index("task")
             task = prot_parts[idx + 1]
             return task
         except (ValueError, IndexError):
-            # default to entire protocol name?
-            # This should actually check the results of other guesses
-            # (like modality) to determine a better default than nothing.
-            # At least parts of the protocol name that were already matched elsewhere
-            # should be excluded
             return None
-    else:
-        # default to entire protocol name?
-        # see above
-        return None
+    return None
 
 
 
 def _guess_run(dcm_dict, record):
+
+	# start with 1.
         run = 1
-        # vielleicht andere Variablen-Namen?
         protocol = record.get("ProtocolName")
+
         # the bold-tasks don't have runs.
         if "DTA_ep2d_bold_Task".lower() in protocol.lower():
               return "1"
         if "DTA_epi_pmu_rest".lower() in protocol.lower():
               return "1"
 
+	# I return 60dir as a run, because it is easy
+	# to change that later to acq-60dir
```

Comment:
This "hack" to abuse the run value isn't necessary. The dictionary you return at the end can be enhanced by any term.
It can contain "bids-acquisition" the same way it contains "bids-run" or "bids-task". The converter will understand that and generate that "acq-60dir" part in the file name.